jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -702,6 +702,13 @@ private InterpreterResult executeSql(String propertyKey, String sql,
               String results = getResults(resultSet,
                   !containsIgnoreCase(sqlToExecute, EXPLAIN_PREDICATE));
               interpreterResult.add(results);
+              try {
+                if (resultSet.isClosed()) {
+                  break;
+                }
+              } catch (Exception e) {
```

Comment:
Yes, agreed, but until `java.sql.SQLException: ERROR 1101 (XCL01): ResultSet is closed.
` is fixed for Phoenix or `isClosed()` implemented in Hive, it's a conscious choice that I think we should take. Because for me it happens all the time;

<img width="1431" alt="screen shot 2017-07-08 at 9 26 31 am" src="https://user-images.githubusercontent.com/674497/27982481-a952ba1a-63bf-11e7-8788-d9cbc0f6d810.png">


But, yes it is expensive. Any other thought?